index.html

```diff
@@ -76,16 +77,16 @@
 			<div class="collapse navbar-collapse navHeaderCollapse">
 				<ul class="nav navbar-nav navbar-right">
 					<li ng-class="{active: $state.includes('speaker-list')}">
-						<a ui-sref="speaker-list">Taleliste</a>
+						<a ui-sref="speaker-list">Visning av taleliste</a>
```

Comment:
Er ikke det selvforklarende? Redd menyen blir for bred på mindre skjermer/oppløsninger med lengre tittel.